tensorboard/webapp/metrics/store/metrics_reducers_test.ts

```diff
@@ -2324,6 +2324,57 @@ describe('metrics reducers', () => {
     });
   });
 
+  describe('metricsCardFullSizeToggled', () => {
+    it('expands card', () => {
+      const state = buildMetricsState();
+      const action = actions.metricsCardFullSizeToggled({
+        cardId: 'card1',
+      });
+      const nextState = reducers(state, action);
+      expect(nextState.cardStateMap).toEqual({
+        card1: {fullWidth: true, tableExpanded: true},
+      });
+    });
+
+    it('expands card when table has expanded', () => {
```

Comment:
nit: expands card when table is already expanded
```suggestion
    it('expands card when table is already expanded', () => {
```